packages/c/centurion/xmake.lua

```diff
@@ -60,7 +59,7 @@ package("centurion")
 
     on_test(function (package)
         assert(package:check_cxxsnippets({ test = [[
-            void test(int args, char **argv) {
+            int main() {
```

Comment:
please use `void test()`

Reply:
I had ``void test`` but some checks failed so @star-hengxing changed it https://github.com/xmake-io/xmake-repo/commit/ffe9235e3e8038b7fe67d36bd323da0e2db9bcb0